### PR TITLE
Ability to controll slides by admin

### PR DESCRIPTION
**Pull request by kdzwinel**
Admins can change slides with arrow keys. Users don't:

screen shot 2016-02-15 at 17 15 37

### CODE REVIEW

File: js/components/slideList.js [hunk 1]
```diff
@@ -25,9 +26,21 @@ class SlideList extends React.Component{
...
-        this.initQuestionRepository();
+    }
+
+    initSlideController() {
+        let sc = new SlideController(this.firebaseRef);
+
+        sc.onSlideChange(this.goToSlide.bind(this));
+
+        //TODO refactor
+        window.addEventListener('keydown', (ev) => {
```
Comment (kdzwinel) [inline js/components/slideList.js]: @matzimowski

**Comment by matzimowski** [comment]
love it :heart: 
:shipit:

**Comment by matzimowski** [comment]
:+1:

**Comment by kdzwinel** [comment]
Is it OK to keep the `window.addEventListener('keydown'` where it is? You used reac bindings for all other events (e.g. clicks).

**Comment by matzimowski** [comment]
@kdzwinel leave it like it is, i do not want to block you, ship it, i will refactor it later if needed.

**Pull request merged**